sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts

```diff
@@ -580,4 +581,30 @@ export class EducationProgramOfferingInstitutionsController extends BaseControll
       throw error;
     }
   }
+
+  /**
+   * Gets the offering simplified details, not including, for instance,
+   * validations, approvals and extensive data.
+   * Useful to have an overview of the offering details, for instance,
+   * when the user needs need to have quick access to data in order to
+   * support operations like confirmation of enrolment or scholastic
+   * standing requests or offering change request.
+   * @param offeringId offering.
+   * @param locationId offering location.
+   * @returns offering details.
+   */
+  @HasLocationAccess("locationId")
+  @ApiNotFoundResponse({
+    description: "Not able to find the Education Program offering.",
+  })
+  @Get("location/:locationId/offering/:offeringId/summary-view")
```

Comment:
IMO word "view" can be avoided on API perspective. "summary" alone or "summary-data" or "summary-details"?

Reply:
The view was not targeting the UI but I can see the confusion.
I would prefer `summary-details` because we use `summary` sometimes for lists.

Reply:
Changed.